### PR TITLE
Added Records and Records UI

### PR DESCRIPTION
**Pull request by NaveenRajaLaxmana**
Kindly review the records UI for further process.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -49,52 +53,64 @@ const App = () => {
...
-          }}/>
-          <Stack.Screen name='plows-edit' component={AddEditPlow} options={{
-            header:({route}) => <HeaderSimple name={"Edit your plow"}/>,
-            headerShown:true
-          }}/>
-          {/* Checking fork to start development */}
-        </Stack.Navigator>
-      </NavigationContainer>
+        <PlowState>
+          <PaperProvider>
```
Comment (Dinesh1129) [inline App.js:57]: What is PaperProvider?

File: screens/Records/AddEditRecord.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+import tw from 'twrnc'
+import {TextInput} from 'react-native-paper'
+import DropDown from 'react-native-paper-dropdown'
+import DatePicker from 'react-native-date-picker'
+import DatePick from './datepick'
+import StartTime from './startTime'
+import EndTime from './endTime'
+import { MyButton } from '../Tractors/AddEditTractor'
+
+const driverlist = [
```
Comment (Dinesh1129) [inline screens/Records/AddEditRecord.js:12]: Why do we need this default value? Don't we take it form storage?

File: screens/Records/datepick.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { useState,useEffect } from "react";
+import { Alert, Modal, StyleSheet, Text, Pressable, View,TouchableOpacity,SafeAreaView } from "react-native";
+import DatePicker from "react-native-date-picker";
+import tw from 'twrnc'
+
+const DatePick = ({modalVisible,setModalVisible,date,setDate}) => {
+
+  return (
+
```
Comment (Dinesh1129) [inline screens/Records/datepick.js:9]: Avoid empty space always

**Review by Dinesh1129 — APPROVED** [review]

File: App.js (hunk 1 above)
Comment (NaveenRajaLaxmana) [reply App.js:57]: It is a separate UI dependency package for Text input

File: screens/Records/AddEditRecord.js (hunk 2 above)
Comment (NaveenRajaLaxmana) [reply screens/Records/AddEditRecord.js:12]: yes Anna,I have know completed UI only.If the UI is Okay then we can add storage values.

File: screens/Records/datepick.js (hunk 3 above)
Comment (NaveenRajaLaxmana) [reply screens/Records/datepick.js:9]: Okay Done.

**Pull request merged**